lizmap/modules/admin/zones/project_list.zone.php

```diff
@@ -0,0 +1,80 @@
+<?php
+/**
+ * Construct a list of Lizmap projects.
+ *
+ * @author    3liz
+ * @copyright 2011 3liz
+ *
+ * @see      http://3liz.com
+ *
+ * @license    Mozilla Public License : http://www.mozilla.org/MPL/
+ */
+class project_listZone extends jZone
+{
+    protected $_tplname = 'project_list_zone';
+
+    protected function _prepareTpl()
+    {
+        $maps = array();
+        // Get repository data
+        $repository = $this->param('repository');
+
+        $repositories = array();
+        if ($repository != null) {
+            $repositories[] = $repository;
+        } else {
+            $repositories = lizmap::getRepositoryList();
+        }
+
+        foreach ($repositories as $r) {
+            if (jAcl2::check('lizmap.repositories.view', $r)) {
```

Comment:
to avoid multiple nested block, and increase the understanding of the code, it is better here  (because there is no else statement) to use the `continue` keyword:
```
if ( ! jAcl2::check('lizmap.repositories.view', $r)) {
    continue;
}
// ... here the right is ok

```